### ISSUE TITLE
Package CICS resources outside src/main/resources

### ISSUE THREAD

**Issue by holgerknoche** [IBM/cics-bundle-maven#198]
We use a custom Maven plugin to generate program definitions from annotations in the Java classes. Being generated, we would like to store them somewhere in target/generated-resources. However, the plugin only allows to specify resource paths in src/main/resources. Could we get an option to also include CICS resources in an arbitrary location inside the project (or at least in target/generated-resources)?

**Comment by ind1go** [comment]
Ah yes, I understand. Using `@CICSProgram` for OSGi bundles is [announced in CICS TS 6.1](https://www.ibm.com/common/ssi/cgi-bin/ssialias?infotype=an&subtype=ca&appname=gpateam&supplier=897&letternum=ENUS222-092#:~:text=videos%2C%20and%20tutorials.-,Java%20annotation%20now%20available%20to%20define%20methods%20to%20link%20to%20OSGi%20Java%20applications,-Java%20applications%20deployed), which will be available soon.

We'll look further into this.

**Comment by ind1go** [comment]
Just to check I have the correct understanding, do you mean the `src/main/bundleParts` directory (instead of `src/main/resources`)? Have you been able to use the [`bundlePartsDirectory`](https://ibm.github.io/cics-bundle-maven/build-mojo.html#bundlePartsDirectory) property?

**Comment by holgerknoche** [comment]
Yes, that's the one. The `bundlePartsDirectory` property does not really help here, because it is restricted to paths within `src/main`. Unlike `target`, these paths are usually under version control, and since we don't want to commit our generated artifacts, we have to manually exclude them.
